packages/search/src/style.scss

```diff
@@ -198,6 +198,23 @@ $input-z-index: 20;
 			display: block;
 		}
 
+		// Removes the fade overlay when the search is open and focused
+		&.has-focus {
```

Comment:
Can't we use the `&:focus` pseudo-selector here?

Reply:
Not really, because it's a div element that's getting the focus, hence this class is used.